spec.bs

```diff
@@ -342,6 +350,27 @@ This is detectable because it can change the set of fields that are read from th
     "slot-size", or "all-slots-requested-sizes", set |interestGroup|'s
     [=interest group/trusted bidding signals slot size mode=] to
     |group|["{{GenerateBidInterestGroup/trustedBiddingSignalsSlotSizeMode}}"].
+  1. Let |adSizes| be a new [=map=] whose [=map/keys=] are [=strings=] and [=map/values=] are
+     [=ad sizes=].
+  1. If |group|["{{GenerateBidInterestGroup/adSizes}}"] [=map/exists=]:
+    1. [=map/For each=] |sizeName| → |size| of |group|["{{GenerateBidInterestGroup/adSizes}}"]:
+      1. If |sizeName| is "", [=exception/throw=] a {{TypeError}}.
+      1. Let |parsedSize| be the result from running [=parse an AdRender ad size=] with |size|.
+      1. If |parsedSize| is null, [=exception/throw=] a {{TypeError}}.
+      1. [=map/Set=] |adSizes|[|sizeName|] to |parsedSize|.
+    1. Set |interestGroup|'s [=interest group/ad sizes=] to |adSizes|.
+  1. Let |sizeGroups| be a new [=map=] whose [=map/keys=] are [=strings=] and [=map/values=]

```

Comment:
`...whose [=map/keys=] are [=strings=] and [=map/values=] are [=strings=].`

Aren't `sizeGroup` values lists of strings...

```
        "adSizes": {
            "MedRect": {"width": "300", "height": "250"},
            "Leaderboard": {"width": "728", "height": "90"}
        },
        "sizeGroups": {
            "mr_lb": ["MedRect","Leaderboard"]
        },
```

Reply:
Thanks for catching this.